fastlane_bot/events/utils.py

```diff
@@ -1341,13 +1341,27 @@ def get_start_block(
     elif mgr.tenderly_fork_id:
         # connect to the Tenderly fork and get the latest block number
         from_block = mgr.w3_tenderly.eth.block_number
+        # Log all non-integer block numbers
+        non_int_values = [
+            (index, block["last_updated_block"], type(block["last_updated_block"]))
+            for index, block in enumerate(mgr.pool_data) if type(block["last_updated_block"]) is not int
+        ]
+        if non_int_values:
+            mgr.cfg.logger.info(f"get_start_block (tenderly {mgr.tenderly_fork_id}): {non_int_values}")
```

Comment:
logs should have this notation [events.utils.get_start_block] for tracking

mgr.cfg.logger.info(f"[events.utils.get_start_block] non_int_values: {non_int_values}")